test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2575,6 +2575,23 @@ def forward(self, x):
         x = torch.randn(2, 3, 4)
         self.run_test(TensorFactory(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_eye(self):
+        class TensorFactory(torch.nn.Module):
+            def forward(self, x):
+                return torch.eye(x.size()[1], 3), torch.eye(4, 4, dtype=torch.long), torch.eye(x.size()[1], 2, dtype=torch.long)
+
+        x = torch.randn(2, 3, 4)
+        self.run_test(TensorFactory(), x)
+
+        class TensorFactoryScript(torch.jit.ScriptModule):
+            @torch.jit.script_method
```

Comment:
Also, if both nn.Module and ScriptModules are the same, you can have one module and enable scripting test using ``@enableScriptTest`` decorator.

Reply:
thx, good advice :)